core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/StreamletPort.scala

```diff
@@ -61,6 +61,18 @@ trait CodecInlet[T] extends Inlet {
    * Describes the schema used to deserialize the data.
    */
   def schemaAsString: String
+
+  /**
+   * Returns if this inlet uses a unique group id.
+   */
```

Comment:
Can we please use the same, more informative comment as used for the Java version below?

Reply:
The one returns the boolean, the other sets it, but yes, I'll add it.